config/environments/development.rb

```diff
@@ -68,5 +68,6 @@
 
   # Use an evented file watcher to asynchronously detect changes in source code,
   # routes, locales, etc. This feature depends on the listen gem.
-  config.file_watcher = ActiveSupport::EventedFileUpdateChecker
+  # Commented out for development purposes with Apple M1 Silicon Procesing Chips
+  # config.file_watcher = ActiveSupport::EventedFileUpdateChecker
```

Comment:
then this needs to be un-commented for people without the M1 chip? (like me) if so, can a reference to do so be added to the repo-readme for the start up instructions?